src/runtime_src/core/pcie/tools/xbutil/xbutil.h

```diff
@@ -360,15 +360,18 @@ class device {
     uint32_t parseComputeUnitStat(const std::vector<std::string>& custat, uint32_t offset, cu_stat kind) const
     {
        uint32_t ret = 0;
+       uint32_t idx = 0;
 
        if (custat.empty())
           return ret;
 
        for (auto& line : custat) {
            uint32_t ba = 0, cnt = 0, sta = 0;
-           std::sscanf(line.c_str(), "CU[@0x%x] : %d status : %d", &ba, &cnt, &sta);
+           ret = std::sscanf(line.c_str(), "CU[@0x%x] : %d status : %d", &ba, &cnt, &sta);
+           if (ret)
+               idx++;
 
-           if (offset != ba)
+           if (offset != ba && offset != idx)
```

Comment:
Please help the next developer with the order of operation:
` if ((offset != ba) && (offset != idx))`

Reply:
done